features/admin.feature

```diff
@@ -73,3 +73,8 @@ Feature: Admin
     When I create a user
     Then I see alert "Bruker registrert!"
 
+  Scenario: Count number of votes and active users
+    Given There is an active election
+    And The election has votes
+    And I am on the edit election page
+    Then I should see number of votes and active users
```

Comment:
Wouldn't it be better to be explicit than to have "magick" numbers in the step definition?

```gherkin
Then I should see 1 in voted count
And I should see 2 in active user count
```